docs/ops/detection/PriorBox_1.md

```diff
@@ -40,31 +74,31 @@
   * **Range of values**:
     * false or 0 - clipping is not performed
     * true or 1 - each value in the output tensor is clipped to [0,1] interval.
-  * **Type**: boolean
+  * **Type**: `boolean`
   * **Default value**: false
   * **Required**: *no*
 
 * *step*
 
   * **Description**: *step* is a distance between box centers. For example, *step* equal 85 means that the distance between neighborhood prior boxes centers is 85.
```

Comment:
Please, remove all `For example, *XXX* equals to ...`. These statements are useless in most cases. If they really make sense the update the first sentence in the description to use it.